Makefile

```diff
@@ -100,41 +93,6 @@ common-checks-2:
 copyright:
 	python scripts/check_copyright.py
 
-.PHONY: check-copyright
```

Comment:
All the below are duplications and also don't use tox. Going forward let's just use tox to have max consistency.